### PR TITLE
Exercise: Recursion

### PR DESCRIPTION
**Pull request by artem-konovalov-personal**
Closed [#204](https://github.com/nesterone/sf-eng-init-gl-spring-2016/issues/204)

### CODE REVIEW

**Comment by artem-konovalov-personal** [comment]
I have written the code, which, as I mean, helps us to resolve the problem with negative values in the `isEven` function and I have added it into the comment block

File: 01-syntax/02-functions-variables-scope-and-memory/isEven.js [hunk 1]
```diff
@@ -4,11 +4,42 @@
...
 
 console.log(isEven(50));
 // → true
 console.log(isEven(75));
 // → false
 console.log(isEven(-1));
-// → ??
+// → Uncaught RangeError: Maximum call stack size exceeded
+
+/* We can to multiply the number to '-1' value.
```
Comment (nesterone) [inline 01-syntax/02-functions-variables-scope-and-memory/isEven.js]: What does all that text means ? 
Please clean up source code, if you have some issues to discuss please start discussion in corresponding issue

**Comment by artem-konovalov-personal** [comment]
I have removed a long comment with additional check's explanations and now there is only one function isEven() in the code

**Comment by artem-konovalov-personal** [comment]
I have changed an unexpected behavior of `isEven()` with negative values and added a couple of additional checks for wrong parameters of this function

**Comment by nesterone** [comment]
Excellent !

**Pull request merged**